serving-api/src/main/scala/com/stratio/sparkta/serving/api/service/http/PolicyContextHttpService.scala

```diff
@@ -126,7 +126,7 @@ trait PolicyContextHttpService extends BaseHttpService {
                     createFragments(fragmentActor, outputs.toList ::: inputs.toList)
                     PolicyResult(policy.id.getOrElse(""), p.name)
                   case Failure(ex: Throwable) => throw new ServingCoreException(ErrorModel.toString(
-                    ErrorModel(ErrorModel.CodeErrorCreatingPolicy, s"Can't create policy")
+                    ErrorModel(ErrorModel.CodeErrorCreatingPolicy, ex.getMessage)
```

Comment:
This error is for the user. I believe it is more useful for the user the old message. If you want you can pass the strackTrace as third argument in the constructor